commands_function.py

```diff
@@ -0,0 +1,59 @@
+import os
```

Comment:
Create file called ICommand 
file called command_factory
and file for each command